styleguide/source/_patterns/02-organisms/topic/topic-rail-right/topic-rail-right.twig

```diff
@@ -0,0 +1,3 @@
+<aside class="topic_rail-right">
```

Comment:
I think Rachel is adding the aside with include in the actual template too, and then also created an organism to show the rail as its own thing?